gold-phone-input.html

```diff
@@ -88,7 +88,7 @@
       <label hidden$="[[!label]]">[[label]]</label>
 
       <div class="horizontal layout">
-        <span>+<span>[[countryCode]]</span></span>
+        <span hidden="[[noCountryCode]]">+<span>[[countryCode]]</span></span>
```

Comment:
Can we actually just wrap this in a div, and add a mixin or a custom property, and do this in CSS? `hidden` needs to be polyfilled for IE11, and adding an extra property doesn't seem worth it in this case (since it's not actually used in the code anywhere)